lib/datadog/tracing/contrib/rails/runner.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Rails
+        # Limit the maximum size of the source code captured in the source tag.
+        MAX_TAG_VALUE_SIZE = 4096
```

Comment:
Can we move these definition under `Runner` module?

Reply:
Done!